src/coreclr/scripts/superpmi.py

```diff
@@ -1722,6 +1722,11 @@ def replay_with_asm_diffs(self):
         altjit_asm_diffs_flags = target_flags
         altjit_replay_flags = target_flags
 
+        option_flags = []
+        if self.coreclr_args.jitoption:
+            for o in self.coreclr_args.jitoption:
```

Comment:
I'm not fond of the user model here, and the asymmetry that you can specify options for the base but not diff.

I would suggest having options `-jitoption_base` and `-jitoption_diff`, only for the asmdiffs command, so you can support passing these to either base or jit or both. SuperPMI itself only has `-jitoption` and `-jitoption2` because it doesn't associate any semantics between the JITs, but superpmi.py does have a semantic associated with the order we pass JITs to superpmi.exe.

Comments?

Reply:
Good idea; done. Though for some reason I used `-base_jit_option` and `-diff_jit_option`.